src/components/layouts/WalletLayout/components/WalletHeader/WalletHeader.js

```diff
@@ -4,14 +4,14 @@ import { css } from 'aphrodite/no-important'
 import tx from '../../../../../theme/styles'
 import sx from './styles'
 
-const displayName = 'components/layouts/DashboardLayout/components/DashboardHeader'
+const displayName = 'components/layouts/WalletLayout/components/WalletHeader'
```

Comment:
while in WalletLayout components dir, `Header` would be a better name. same applies for WalletNav